src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java

```diff
@@ -0,0 +1,24 @@
+package edu.cornell.kfs.gl.service.impl.fixture;
+
+public enum EmailAdressTestValue {
+    THREE_PART_DOMAIN_GOOD("ww5@a.b.c.org", true),
+    ALPHANUMERIC_DOMAIN_DOT_ADDRESS_GOOD("something.else@a2.com", true),
+    ALPHANUMERIC_DOMAIN_UNDERSCORE_ADDRESS_GOOD("something_else@a2.com", true),
+    ALPHANUMERIC_DOMAIN_HYPHEN_ADDRESS_GOOD("something-else@a2.com", true),
+    UNDERSCORE_ADDRESS_GOOD("something_else@something.else.com", true),
+    HYPHEN_DOCMAIN_GOOD("something-else@et-tu.com", true),
```

Comment:
I think there is a typo here. Did you intend this to be HYPHEN_DOMAIN_GOOD ?

Reply:
fixed